### DISCUSSION TITLE
How to implement PP-Structurev2 (PP-PicoDet) for table detection in a document

### DISCUSSION THREAD

**Discussion by arhamshah** [PaddlePaddle/PaddleOCR#7717] [Q&A]
I want to perform layout analysis for given below image to detect all the tables. I searched your documentation but didn't find any such quick start code. I want to implement PP-PicoDet for this task, can anyone help me with the code part for the same.
![template](https://user-images.githubusercontent.com/25555221/191941571-827eb1a7-f905-4cff-8d6a-9dbd426beadf.PNG)

**Comment by an1018** [comment]
You can refer to[ layout-analysis document](https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/ppstructure/layout/README.md),  and modify the ym configuration file, for example dataset path.

**Reply by TapendraBaduwal** [reply]
@arhamshah 
@an1018 
Dear @arhamshah, @an1018 
How to recognition multiple languages text in the same cell coordinates of table? It is possible to use language classification model in this case?
